media-libs/raylib/raylib-1.9.7.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-multilib
```

Comment:
Is there a specific necessity to make this multilib? Unless you have 32-bit binary packages depending on this, multilib is just an unnecessary complexity.